javascript/builtins/intl/NumberFormat.json

```
@@ -571,40 +571,88 @@
             },
             "options_useGrouping_parameter": {
               "__compat": {
-                "description": "<code>options.useGrouping</code> parameter",
                 "support": {
                   "chrome": {
-                    "version_added": false
+                    "version_added": "24"
                   },
                   "chrome_android": "mirror",
                   "deno": {
-                    "version_added": false
+                    "version_added": "1.8"
+                  },
+                  "edge": {
+                    "version_added": "12"
                   },
-                  "edge": "mirror",
                   "firefox": {
-                    "version_added": "preview"
+                    "version_added": "29"
                   },
-                  "firefox_android": "mirror",
-                  "ie": {
-                    "version_added": false
+                  "firefox_android": {
+                    "version_added": "56"
                   },
-                  "nodejs": {
-                    "version_added": false
+                  "ie": {
+                    "version_added": "11"
                   },
+                  "nodejs": [
+                    {
+                      "version_added": "13.0.0"
+                    },
+                    {
+                      "version_added": "0.12.0",
+                      "partial_implementation": true,
+                      "notes": "Before version 13.0.0, only the locale data for <code>en-US</code> is available by default. When other locales are specified, the <code>NumberFormat</code> instance silently falls back to <code>en-US</code>. To make full ICU (locale) data available before version 13, see <a href='https://nodejs.org/docs/latest/api/intl.html#intl_options_for_building_node_js'>Node.js documentation on the <code>--with-intl</code> option</a> and how to provide the data."
+                    }
+                  ],
                   "opera": "mirror",
                   "opera_android": "mirror",
                   "safari": {
-                    "version_added": false
+                    "version_added": "10"
                   },
                   "safari_ios": "mirror",
                   "samsunginternet_android": "mirror",
                   "webview_android": "mirror"
                 },
                 "status": {
-                  "experimental": true,
+                  "experimental": false,
                   "standard_track": true,
                   "deprecated": false
                 }
+              },
+              "options_useGrouping_parameter-string_values": {
```

Since this is a subfeature of `options_useGrouping_parameter`, we can simplify the name:

```suggestion
              "string_values": {
```